addons/event/models/event.py

```diff
@@ -271,10 +274,9 @@ def _onchange_type(self):
 
             if self.event_type_id.event_type_mail_ids:
                 self.event_mail_ids = [(5, 0, 0)] + [(0, 0, {
-                    'template_id': line.template_id,
-                    'interval_nbr': line.interval_nbr,
-                    'interval_unit': line.interval_unit,
-                    'interval_type': line.interval_type})
+                            attribute_name: getattr(line, attribute_name)
+                            for attribute_name in self.env['event.type.mail']._get_common_event_mail_fields()
```

Comment:
meh. Not sure about the naming.. maybe `_get_event_mail_fields_whitelist()` ?

Reply:
Ok, just for you!

Reply:
<3